app/controllers/order/search_code.js

```diff
@@ -0,0 +1,125 @@
+import Ember from 'ember';
+import { translationMacro as t } from "ember-i18n";
+import AjaxPromise from 'browse/utils/ajax-promise';
+
+const { getOwner } = Ember;
+
+export default Ember.Controller.extend({
```

Comment:
I think we can avoid lot of lines if we extend this controller from `search_module`.
What do you think??

Reply:
Okay, Let me check. Thanks for flagging this. :)

Reply:
Good suggestion but Currently we do not have search_module in browse to extend from it. We have that in stock so we need to move that to goodcity-lib first then we can use that for both stock and browse.